src/conf.py

```diff
@@ -29,10 +32,27 @@
 # Add any Sphinx extension module names here, as strings. They can be
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
 # ones.
-extensions = ['sphinx.ext.graphviz']
+extensions = [
+    'sphinx.ext.graphviz',
+    'sphinx.ext.autodoc',
+    'sphinx.ext.doctest',
+    'sphinx.ext.todo',
+    'sphinx.ext.coverage',
+    'sphinx.ext.mathjax',
+    'breathe'
```

Comment:
Is there any reason to keep the dependency on `breathe` (which pull doxygen into sphinx)?  
If not, now might be a good time to remove it.

Reply:
5 seconds of googling showed there is a way to add breathe to RTD but I vote to remove it until we decide its worth doing.

Reply:
I don' think we need it. Using breathe to pull doxygen into sphinx results in source code docs that are less useful; i.e., not as easily navigable, etc.

Reply:
I'll take out loading `breathe` and all lines having to do with it.